packages/camera/camera_platform_interface/pubspec.yaml

```diff
@@ -3,22 +3,22 @@ description: A common platform interface for the camera plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/camera/camera_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.0-nullsafety
+version: 2.0.0
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0-nullsafety.6
-  plugin_platform_interface: ^1.1.0-nullsafety.2
-  cross_file: ^0.3.0-nullsafety
-  stream_transform: ^2.0.0-nullsafety.0
+  meta: ^1.3.0
+  plugin_platform_interface: ">=1.0.0 <3.0.0"
+  cross_file: ^0.3.0
```

Comment:
I just pushed a nullability fix to cross_file; since this hasn't landed yet it would be good to bump this to 0.3.1 just to help ensure that people are getting that fixed version.

Reply:
Done, thanks for the heads-up